packages/@lwc/engine/src/framework/hot-swaps.ts

```diff
@@ -0,0 +1,213 @@
+import { VM, scheduleRehydration } from './vm';
+import { isFalse, isUndefined, isNull } from '@lwc/shared';
```

Comment:
Move bare modules imports above local imports.

Reply:
what is this?

Reply:
The imports have to be sorted by the module type. external module go above local imports. We should probably have a lint rule for this.
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#importorder-enforce-a-convention-in-module-import-order

Reply:
is there a reason for such order? in fact, because the order does matter, I will argue that if anything, it should be the other way around :)

Reply:
I will fix it, but I don't agree with such stylistic choice! just saying!

Reply:
Import order does have an impact on how the code gets evaluated, we already faced a similar issue in the past with prototype patching. If the `@lwc/shared` were to patch constructors we would run in a similar issue. IMO, keeping the import order between the different modules makes the evaluation more predictable.